### PR TITLE
web: fix redirection define password

### PR DESCRIPTION
**Pull request by maxime-carabina**
We are fixing link between steps and define password screens.

### CODE REVIEW

File: web-frontend/src/pages/CreateAccount/StepOne.tsx [hunk 1]
```diff
@@ -1,37 +1,84 @@
...
+          <div className="flex flex-col justify-center content-center items-center w-fit h-fit max-w-sm">
+            <div className="w-full max-w-xs mb-6">
+              <Stepper
+                step={0}
+                steps={[
+                  Intl.t('account.create.step1.title'),
+                  Intl.t('account.create.step2.title'),
+                  Intl.t('account.create.step3.title'),
+                ]}
+              />{' '}
```
Comment (Thykof) [inline web-frontend/src/pages/CreateAccount/StepOne.tsx:61]:
```suggestion
              />
```

**Review by Thykof — CHANGES_REQUESTED** [review]

**Review by Thykof — APPROVED** [review]

File: web-frontend/src/i18n/en_US.json [hunk 2]
```diff
@@ -5,7 +5,34 @@
     "title_third_part": "wallet"
   },
   "account": {
```
Comment (mazmassa) [inline web-frontend/src/i18n/en_US.json:7]: Beautiful folks! 👏 @Thykof @maxime-carabina

File: web-frontend/src/i18n/en_US.json [hunk 3]
```diff
@@ -5,7 +5,34 @@
...
+      "input": {
+        "nickname": "Enter an account name"
+      }
+    },
+    "import": "Import an existing account",
+    "add": "Add an account",
+    "select": "Select an account"
+  },
+  "errors": {
+    "required_nickname": "Nickname is required"
```
Comment (mazmassa) [inline web-frontend/src/i18n/en_US.json:36]:
```suggestion
    "required-nickname": "Nickname is required"
```

File: web-frontend/src/i18n/en_US.json [hunk 4]
```diff
@@ -5,7 +5,34 @@
...
+      },
+      "step2": {
+        "title": "Password",
+        "description": "Now, define your password. To maximize security, a pop up will open directly on your device via MassaStation."
+      },
+      "step3": {
+        "title": "Backup"
+      },
+      "buttons": {
+        "define_password": "Define a password",
```
Comment (mazmassa) [inline web-frontend/src/i18n/en_US.json:24]:
```suggestion
        "define-password": "Define a password",
```

File: web-frontend/src/pages/CreateAccount/StepOne.tsx [hunk 5]
```diff
@@ -1,37 +1,84 @@
...
+  const form = useRef(null);
+
+  const [error, setError] = useState<IErrorObject | null>(null);
+
+  function validate(e: SyntheticEvent) {
+    const formObject = parseForm(e);
+    const { nickname } = formObject;
+
+    if (!nickname || !nickname.length) {
+      setError({ nickname: Intl.t('errors.nickname_required') });
```
Comment (mazmassa) [inline web-frontend/src/pages/CreateAccount/StepOne.tsx:26]:
```suggestion
      setError({ nickname: Intl.t('errors.nickname-required') });
```

File: web-frontend/src/pages/CreateAccount/StepOne.tsx [hunk 6]
```diff
@@ -1,37 +1,84 @@
...
+  const navigate = useNavigate();
+  const form = useRef(null);
+
+  const [error, setError] = useState<IErrorObject | null>(null);
+
+  function validate(e: SyntheticEvent) {
+    const formObject = parseForm(e);
+    const { nickname } = formObject;
+
+    if (!nickname || !nickname.length) {
```
Comment (mazmassa) [inline web-frontend/src/pages/CreateAccount/StepOne.tsx:25]: Non blocking: only one of them is not enough!?

**Review by mazmassa — APPROVED** [review]
Once fixed the comments, LGTM! Thanks for that @Thykof @maxime-carabina 

👏

File: web-frontend/src/i18n/en_US.json (hunk 3 above)
Comment (maxime-carabina) [reply web-frontend/src/i18n/en_US.json]: Done

File: web-frontend/src/i18n/en_US.json (hunk 4 above)
Comment (maxime-carabina) [reply web-frontend/src/i18n/en_US.json:24]: Done

File: web-frontend/src/pages/CreateAccount/StepOne.tsx (hunk 6 above)
Comment (maxime-carabina) [reply web-frontend/src/pages/CreateAccount/StepOne.tsx]: Done

**Pull request merged**